doc/gui/examples/matplotlib_example.py

```diff
@@ -0,0 +1,22 @@
+# Integrating Matplotlib into Taipy using figure property
+
+# Import Necessary Libraries:
+from taipy.gui import Gui 
+import taipy.gui.builder as tgb
+import matplotlib.pyplot as plt  
+from matplotlib.figure import Figure 

```

Comment:
Is `Figure` used in the code? Why do you import it?

Reply:
Right sir, the matplotlib.figure.Figure property is not being used here. Will make the changes in updated code. Thank you sir.